upstream-community-operators/starboard-operator/0.0.1/starboard-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,316 @@
+      - bar
+      - baz
+    EOF
+    ```
+
+    ### AllNamespaces
+
+    Configure the operator to watch for events in all namespaces:
+
+    ```
+    cat << EOF | kubectl apply -f -
+    apiVersion: operators.coreos.com/v1alpha2
+    kind: OperatorGroup
+    metadata:
+      name: starboard
+      namespace: operators
+    spec: {}
+    EOF
+    ```
+
+    ## Install operator
+
+    To install the operator create a new Subscription to the selected channel for the Starboard Operator.
+    OLM uses this information to create the ClusterServiceVersion and eventually start the operator pod.
+
+    ```
+    cat << EOF | kubectl apply -f -
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: starboard-operator
+      namespace: operators
+    spec:
+      channel: alpha
+      name: starboard-operator
+      source: upstream-community-operators
+      sourceNamespace: marketplace
+    EOF
+    ```
+
+    ## Vulnerability scanners
+
+    By default the operator uses [Trivy](https://github.com/aquasecurity/trivy) to find security vulnerabilities in
+    container images. However, you can configure it to use the Aqua CSP scanner instead.
+
+    Start by creating the `starboard-operator` secret in the `operators` namespace, which holds Aqua CSP config
+    keys to connect to the management console:
+
+    ```
+    $ kubectl create secret generic starboard-operator \
+      --namespace operators \
+      --from-literal OPERATOR_SCANNER_AQUA_CSP_USERNAME=$AQUA_CONSOLE_USERNAME \
+      --from-literal OPERATOR_SCANNER_AQUA_CSP_PASSWORD=$AQUA_CONSOLE_PASSWORD \
+      --from-literal OPERATOR_SCANNER_AQUA_CSP_VERSION=$AQUA_VERSION \
+      --from-literal OPERATOR_SCANNER_AQUA_CSP_HOST=http://csp-console-svc.aqua:8080
+    ```
+
+    Create a new Subscription to the selected channel for the Starboard Operator and enable Aqua CSP scanner via
+    the `config` filed in the Subscription object:
+
+    ```
+    cat << EOF | kubectl apply -f -
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: starboard-operator
+      namespace: operators
+    spec:
+      channel: alpha
+      name: starboard-operator
+      source: upstream-community-operators
+      sourceNamespace: marketplace
+      config:
+        env:
+        - name: OPERATOR_SCANNER_TRIVY_ENABLED
+          value: "false"
+        - name: OPERATOR_SCANNER_AQUA_CSP_ENABLED
+          value: "true"
+        envFrom:
+        - secretRef:
+            name: starboard-operator
+    EOF
+    ```
+
+  keywords: ['aqua-security', 'scanning', 'security']
+  maintainers:
+    - name: Daniel Pacak
+      email: daniel.pacak@aquasec.com
+  provider:
+    name: Aqua Security, Inc.
+  maturity: alpha
+  labels:
+    name: starboard-operator
+  selector:
+    matchLabels:
+      name: starboard-operator
+  links:
+    - name: Starboard Operator on GitHub
+      url: https://github.com/aquasecurity/starboard-operator
```

Comment:
I guess this can be removed since we've agreed to bring the operator into the main repo

Reply:
Good spot. Deleted.